scripts/scripts.js

```diff
@@ -27,6 +27,48 @@ function buildHeroBlock(main) {
   }
 }
 
+function createIcon(name) {
+  const icon = document.createElement('span');
+  icon.classList.add('icon', `icon-${name}`);
+
+  return icon;
+}
+
+function buildAccordionBlock(main) {
```

Comment:
That method should go into the `accordion.js`

Reply:
Fixed